server/server.js

```diff
@@ -339,19 +343,16 @@ app.post('/', jsonBodyParser, async (req, res) => {
             if(button === null) {
                 log(`Bad request: UUID is not registered. UUID is ${req.body.UUID}`);
                 res.status(400).send();
-            }
-            else {
-                let numPresses = 1
+            } else {
+                await handleValidRequest(button, 0.5)
 
-                if(req.body.Type == 'double_click') {
-                    numPresses = 2;
+                if (req.body.Type == 'double click') {
```

Comment:
If things can sometimes be a bit iffy, why not req.body.Type.startsWith('double') so it's caught no matter what?

Reply:
Thanks @darvell , good idea. 

Done!